frontend/src/i18n/config.ts

```diff
@@ -4,6 +4,25 @@ import { initReactI18next } from 'react-i18next';
 
 const en = {}; // To keep TS happy.
 
+export const supportedLanguagesWithLabels = [
+  {
+    id: 'en',
+    label: 'English',
+  },
+  {
+    id: 'pt',
+    label: 'Português',
+  },
+  {
+    id: 'es',
+    label: 'Español',
+  },
+  {
+    id: 'fr',
+    label: 'Français',
+  },
+];
```

Comment:
It can give us the name of the language in one of the specified language.
Like either all in English and not in there native languages. That's why we need this custom mapping for the native language name for code